kernel/FE/BaseBasisFunction.h

```diff
@@ -145,6 +145,11 @@ class BaseBasisFunction {
         return 0;
     }
 
+    virtual LinearAlgebra::SmallVector<1> evalCurl(
+        const Geometry::PointReference<1>& p) const {
+        return {evalDeriv0(p)};
```

Comment:
is that not consistent with the other evalCurl below?

Reply:
What is not consistent, the signature or the implementation?

Because the signature matches that of evalCurl for 2/3D in BaseBasisFunction. The difference here is that these values are computed relatively infrequently.

Reply:
I mean in the other dimensions you throw an assert on evalCurl.

Reply:
Hmm, may indeed be a better approach.